Source/Tests/TrackableEntities.EF.5.Tests/FamilyDbContextTests.cs

```diff
@@ -72,6 +72,27 @@ public void Apply_Changes_Should_Mark_Modified_Parent_Properties()
         }
 
         [Fact]
+        public void Apply_Changes_Should_Skip_NotMapped_Properties()
+        {
+            // Arrange
+            var context = TestsHelper.CreateFamilyDbContext(CreateFamilyDbOptions);
+            var child = new Child("Child")
+            { 
+                Nickname1 = "Tony",
+                Nickname2 = "Sneed",
+                TrackingState = TrackingState.Modified,
+                ModifiedProperties = new HashSet<string>(new[] { nameof(Child.Nickname1), nameof(Child.Nickname2) })
+            };                                                                           
+
+            // Act
+            context.ApplyChanges(child);
+
+            // Assert
+            // Success - no exception has been thrown.
+            // In the future, let's consider marking state to unchanged if modified properties only contained unmapped-properties
+        }         
```

Comment:
You need to have the test fail without the feature implemented.  So if you want to change the test, first do that in a separate commit, before implementing the fix, so that you have a proper failing test.